src/Google.Api.Gax.Grpc/ClientHelper.cs

```diff
@@ -86,7 +86,10 @@ public ClientHelper(ServiceSettingsBase settings)
             where TResponse : class, IMessage<TResponse>
         {
             CallSettings baseCallSettings = _clientCallSettings.MergedWith(perMethodCallSettings);
+            // These operations are applied in reverse order.
+            // I.e. Version header is added first, then retry is performed.
```

Comment:
It's not really clear to me what this means. I would expect the two things to be basically orthogonal - we're only setting up what can happen when we execute the call, aren't we? If the methods were called in the opposite order, what difference would there be?

Reply:
You're right, these are orthagonal, and changing the order would make no difference.
The comment is there just to be consistent with the `BuildApiCall` method above.